client/src/components/layout/TopBar.js

```diff
@@ -14,7 +14,17 @@ const TopBar = ({ user }) => {
     </li>,
   ];
 
+  let id = ""
+  if(user){
+    id = user.id
+  }
```

Comment:
this is okay, but in the future use optional chaining
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
i couldn't figure out the option changing, ill look into more later